static/js/services/live-list.js

```diff
@@ -107,11 +107,12 @@ angular.module('inboxServices').factory('LiveListConfig',
           var form = _.findWhere($scope.forms, { code: report.form });
           scope.route = 'reports';
           scope.icon = form && form.icon;
-          scope.heading = report.contact || report.from;
+          scope.heading =
+            report.valid_subject ? report.subject.value : $translate.instant('report.subject.unknown');
           scope.date = report.reported_date;
           scope.summary = form ? form.title : report.form;
           scope.showStatus = true;
-          scope.valid = report.valid;
+          scope.valid = report.valid && report.valid_subject;
```

Comment:
Having an unknown subject doesn't necessarily make this report invalid. We rely on sentinel validation rules to set the `valid` property, so I don't think this is necessary.

Reply:
Ok, sure .. from the Issue thread I understood it should be set as "invalid" visually.